teamB/seongmin/westagram/controllers/userController.js

```diff
@@ -0,0 +1,32 @@
+const userService = require('../services/userService');
+
+const signUp = async (req, res) => {
+    try {
+        const { name, email, profile_image, password } = req.body;
+
+        if (!name || !email || !profile_image || !password) {
+            return res.status(404).json({message : "KEY_ERROR"});
```

Comment:
KEY ERROR의 경우에는 Client에서 잘 못된 값을 전달했기 때문에, 404 보다는 400 error가 더 정확합니다.

```suggestion
            return res.status(400).json({message : "KEY_ERROR"});
```

Reply:
status code를 400으로 수정하였습니다.